tests/integration/test_load_and_run_multi_agent_a2a_tool.py

```diff
@@ -203,15 +203,13 @@ def test_load_and_run_multi_agent_a2a_sync(agent_framework: AgentFramework) -> N
 
     Note that there is an issue when using Google ADK: https://github.com/google/adk-python/pull/566
     """
-    if agent_framework in [
-        # async a2a is not supported
-        AgentFramework.SMOLAGENTS,
-        # spans are not built correctly
-        AgentFramework.LLAMA_INDEX,
-        # AgentFramework.GOOGLE,
-    ]:
+    skip_reason = {
+        AgentFramework.SMOLAGENTS: "async a2a is not supported; run_async_in_sync fails",
+        # AgentFramework.LLAMA_INDEX: "spans are not built correctly",
```

Comment:
If this is commented out then can it be removed?

Reply:
As I mentioned, I'd rather leave it until we set the trace tree check back in place and we check if this actually fails.

Reply:
Why is this change related to MCP? If it's not related to your MCP changes, the place to change whether a test should be skipped or not is not correct for this PR.

Reply:
If the comment is out-of-date etc, I would rather it be left out of date and no changes made to this file, and address the change in a future PR. Trying to keep this PR as focused as possible on only MCP changes.

Reply:
Yes, please remove it from this PR